tests/addon-template/package.json

```diff
@@ -33,6 +33,10 @@
   },
   "devDependencies": {
     "@babel/core": "^7.19.3",
+    "@babel/plugin-proposal-decorators": "^7.22.5",
+    "@babel/plugin-transform-class-properties": "7.24.7",
+    "@babel/plugin-transform-private-methods": "7.24.7",
+    "@babel/plugin-transform-private-property-in-object": "^7.24.7",
```

Comment:
This is a non-blocking comment, but it's probably safe for us to replace all four of these with https://github.com/ef4/decorator-transforms instead. It could be a separate PR.